extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagator.java

```diff
@@ -58,7 +58,10 @@ public static OtTracerPropagator getInstance() {
     if (!spanContext.isValid()) {
       return;
     }
-    setter.set(carrier, TRACE_ID_HEADER, spanContext.getTraceIdAsHexString());
+    setter.set(
```

Comment:
Leave a comment explaining we are doing this explicitly as the format only supports 64-bit ids?

Reply:
done